content/documentation/pages/7-resources/3-faq.md

```diff
@@ -417,6 +417,40 @@ See the [compatibility matrix](https://github.com/spring-cloud/spring-cloud-stre
 
 <!--END_QUESTION-->
 
+<!--QUESTION#lifecycle-->
+
+Can I manage binding lifecycles?
+
+By default, bindings are started automatically when the application is initialized.
+Bindings implement the Spring `SmartLifecycle` interface.
+`SmartLifecycle` allows beans to be started in phases.
+Producer bindings are started in an early phase (`Integer.MIN_VALUE + 1000`).
+Consumer bindings are started in a late phase (`Integer.MAX_VALUE - 1000`).
+This leaves room in the spectrum such that user beans implementing `SmartLifecycle` can be started before producer bindings, after consumer bindings, or anywhere in between.
+
+You can disable auto-startup by setting the consumer or producer `autoStartup` property to `false`.
+
+Binding lifecycles can be visualized and controlled using Boot actuators; see [Binding visualization and control](https://cloud.spring.io/spring-cloud-static/spring-cloud-stream/current/reference/html/spring-cloud-stream.html#binding_visualization_control).
+
```

Comment:
Let's link to https://cloud.spring.io/spring-cloud-static/spring-cloud-stream/3.0.0.RELEASE/reference/html/spring-cloud-stream.html#binding_visualization_control as we are working on making `current` point to that.

Reply:
Strike that as it has been done already.

Reply:
I specifically used `current` so the link stays up to date - we don't want to point to old docs when, say, we go to 3.1 etc.

Reply:
Right, but the 'current' didn't work properly until recently.

Reply:
I checked it before I changed it from 3.0.0 😄